### PR TITLE
feat(seo): metadata block handling

### LINKED ISSUE #729: Server side metadata block handling
As discussed in https://github.com/adobe/helix-home/discussions/178, blocks with `metadata` class should be processed on the server side in order to optimize SEO and reduce project code.

Depends on #638.

### PR DESCRIPTION
**Pull request by rofe**
Fix #729

### CODE REVIEW

File: src/html.pre.js [hunk 1]
```diff
@@ -11,6 +11,24 @@
  */
 const { getAbsoluteUrl, wrapContent, toClassName } = require('./utils.js');
 
+function readBlockConfig($block) {
+  if (!$block) return {};
```
Comment (tripodsan) [inline src/html.pre.js:15]: no one liners.... didn't lint complain ?

**Review by tripodsan — APPROVED** [review]
i would also add a rendering init test with markdown

File: src/html.pre.js (hunk 1 above)
Comment (rofe) [reply src/html.pre.js:15]: Nope... maybe we need to adjust `@adobe/eslint-config-helix`? 😉

**Pull request merged**